x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.scss

```diff
@@ -27,36 +27,41 @@
 
 .lnsLayerPanel__dimension {
   @include euiFontSizeS;
-  background: lightOrDarkTheme($euiColorEmptyShade, $euiColorLightestShade);
```

Comment:
I was a bit surprised to see how low-contrast the empty dimensions look now, with gray on gray text. Would you consider choosing a darker text color or lighter background color, or do you think the low contrast is effective here?

<img width="305" alt="Screen Shot 2020-09-30 at 2 13 31 PM" src="https://user-images.githubusercontent.com/666475/94723682-25ef9c00-0327-11eb-9ccc-701331b196da.png">

Reply:
Yes the low contrast here was intentional as an additional support of #76021 and as indicated in the mocks in #75938. It further helps distinguish between filled and unfilled dimensions. The contrast itself is still sufficient to meet a11y. 
<img width="244" alt="Screen Shot 2020-09-30 at 14 21 41 PM" src="https://user-images.githubusercontent.com/549577/94724754-bed2e700-0328-11eb-94a0-4c406006a7f6.png">